vm/chef/cookbooks/elasticsearch/files/elasticsearch-utils

```diff
@@ -94,15 +95,26 @@ function change_user_password() {
 
   local -r user_to_change="$4"
   local -r new_pass="$5"
+  local -r protocol="$6"
```

Comment:
I'd change order of arguments to get protocol and hostname first

Reply:
Done.